Source/Diagnostics/ComputeDiagFunctors/JdispFunctor.cpp

```diff
@@ -0,0 +1,63 @@
+/* This file is part of Warpx.
+ *
+ * Authors: Avigdor Veksler
+ * License: BSD-3-Clause-LBNL
+*/
+#include "JdispFunctor.H"
+
+#include "WarpX.H"
+#include "FieldSolver/FiniteDifferenceSolver/HybridPICModel/HybridPICModel.H"
+#include "Particles/MultiParticleContainer.H"
+
+#include <AMReX.H>
+#include <AMReX_IntVect.H>
+#include <AMReX_MultiFab.H>
+
+JdispFunctor::JdispFunctor (int dir, int lev, amrex::IntVect crse_ratio, int ncomp)
+    : ComputeDiagFunctor(ncomp, crse_ratio), m_dir(dir), m_lev(lev)
+{ }
+
+void
+JdispFunctor::operator() (amrex::MultiFab& mf_dst, int dcomp, const int /*i_buffer*/) const
+{
+    auto& warpx = WarpX::GetInstance();
+    auto* hybrid_pic_model = warpx.get_pointer_HybridPICModel();
+
+    /** pointer to source1 (Ji) multifab */
```

Comment:
Comment should be changed to say total simulation current or something like that (not `Ji`).